src/app/views/sidebar/Sidebar.styles.ts

```diff
@@ -113,7 +110,8 @@ export const sidebarStyles = (theme: ITheme) => {
       fontSize: FontSizes.icon,
       textAlign: 'left',
       verticalAlign: 'center',
-      cursor: 'pointer'
+      cursor: 'pointer',
+      color: 'rgb(0, 120, 212)'
```

Comment:
for color, consider using `theme.palette.themePrimary` or the `theme.palette.blue's` instead to ensure consistency when a user is using dark/high contrast theme and for accessibility compliance as well.. 
The color used isn't compliant with the accessibility colour contrast rules 
![image](https://user-images.githubusercontent.com/18407044/151811895-e54aa943-d683-4c07-98a9-edaf5605d881.png)